packages/ionic/src/commands/start.ts

```diff
@@ -304,6 +304,36 @@ Use the ${input('--type')} option to start projects using older versions of Ioni
     }
   }
 
+  async getProjectType() {
+    if (this.env.flags.interactive) {
```

Comment:
With a default provided, you shouldn't need to do this. You can confirm by removing the `else` and running with `--no-interactive`